kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -147,6 +147,7 @@ def test_do_xcom_push_defaults_false(self):
         old_config_path = get_kubeconfig_path()
         shutil.copy(old_config_path, new_config_path)
         k = KubernetesPodOperator(
+            kubernetes_conn_id=None,
```

Comment:
Doesn’t feel like a good idea to do this. We should instead modify the tests (maybe use mocks) to actually test the conn id.